src/main/webapp/WEB-INF/tags/instructor/courseStudentDetails/studentInformationTable.tag

```diff
@@ -0,0 +1,215 @@
+<%@ tag description="instructorCourseStudentDetails / instructorStudentRecords - Student Information" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ attribute name="studentInfoTable" type="teammates.ui.template.StudentInfoTable" required="true" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ tag import="teammates.common.datatransfer.CommentParticipantType" %>
+<div class="well well-plain">
+    <button type="button" class="btn btn-default btn-xs icon-button pull-right"
+        id="button_add_comment" data-toggle="tooltip"
+        data-placement="top" title="" data-original-title="Add comment"
+        <c:if test="${not studentInfoTable.ableToAddComment}">disabled="disabled"</c:if>>
+        <span class="glyphicon glyphicon-comment glyphicon-primary"></span>
+    </button>
+    <div class="form form-horizontal"
+        id="studentInfomationTable">
+        <div class="form-group">
+            <label class="col-sm-1 control-label">Student Name:</label>
+            <div class="col-sm-11"
+                id="<%=Const.ParamsNames.STUDENT_NAME%>">
+                <p class="form-control-static">${studentInfoTable.name}</p>
+            </div>
+        </div>
+        <c:if test="${studentInfoTable.hasSection}">
+            <div class="form-group">
+                <label class="col-sm-1 control-label">Section Name:</label>
+                <div class="col-sm-11"
+                    id="<%= Const.ParamsNames.SECTION_NAME %>">
+                    <p class="form-control-static">${studentInfoTable.section}</p>
+                </div>
+            </div>
+        </c:if>
+        <div class="form-group">
+            <label class="col-sm-1 control-label">Team Name:</label>
+            <div class="col-sm-11"
+                id="<%= Const.ParamsNames.TEAM_NAME %>">
+                <p class="form-control-static">${studentInfoTable.team}</p>
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="col-sm-1 control-label">Official Email Address:</label>
+            <div class="col-sm-11"
+                id="<%= Const.ParamsNames.STUDENT_EMAIL %>">
+                <p class="form-control-static">${studentInfoTable.email}</p>
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="col-sm-1 control-label">Comments:</label>
+            <div class="col-sm-11"
+                id="<%= Const.ParamsNames.COMMENTS %>">
+                <p class="form-control-static">${studentInfoTable.comments}</p>
+            </div>
+        </div>
+    </div>
+</div>
+<div id="commentArea" class="well well-plain" style="display: none;">
+    <form method="post" action="<%= Const.ActionURIs.INSTRUCTOR_STUDENT_COMMENT_ADD %>" name="form_commentadd">
+        <div class="form-group form-inline">
+            <label style="margin-right: 24px;">Recipient:
+            </label> 
```

Comment:
For better readability, you can do either:

<label style="margin-right: 24px;">
    Recipient:
</label>

or 

<label style="margin-right: 24px;">Recipient: </label>